Cargo.toml

```diff
@@ -31,6 +31,7 @@ tray = [ "tao/tray" ]
 devtools = [ ]
 transparent = [ ]
 fullscreen = [ ]
+header = [ "webkit2gtk/v2_36" ]
```

Comment:
maybe we should call it `linux-headers` or `linux-custom-protocol-headers`?

Reply:
or maybe just make version features similar to webkit2gtk, for example to "audio without user interaction" fix/feature needs 2.36 apis too

Reply:
@FabianLars Do you have the link for audio link as reference?

Reply:
The test branch still exists: https://github.com/tauri-apps/wry/compare/dev...autoplay-0.19
i'll add a link to the docs to this comment once i find it again

Reply:
Oh nevermind it was 2.30...

Reply:
I'll pretend I didn't see that. :see_no_evil: 
@lucasfernog and I want to do some workaround in webkit2gtk and we can just check version during runtime.
After that, I'll switch flatpak and bring to v2 soon.

Reply:
I tried to remove feature flags in webkit2gtk-rs but there are too many. I fear that will cause trouble in future updates. I would like to ask to consider this PR again.

Reply:
Let's do it this way for now. if we need more version-related feature flags, we can either change it or recommend users to add the webkit2gtk dependency and define the feature they need.